src/pages/publication/new_publication/LoadPublication.tsx

```diff
@@ -20,7 +48,27 @@ const LoadPublication: React.FC<LoadPublicationProps> = ({ expanded, onChange })
       expanded={expanded}
       onChange={onChange}
       ariaControls="publication-method-file">
-      {t('publication:publication.load_file_description')}
+      {uppy ? (
+        <>
+          <UppyDashboard uppy={uppy} />
```

Comment:
noen mulighet til å få dette til å funke med USE_MOCK=true ?

Reply:
samme som for files-taben, at vi må mocke fetch uppy bruker for å laste opp fil som ikke var helt rett frem. Dette er enkelt i tester, men har ikke funnet en god måte å gjøre det på for lokal mock